manufacturing-ai-suite/industrial-edge-insights-time-series/docs/user-guide/wind-turbine-anomaly/index.md

```diff
@@ -0,0 +1,99 @@
+# Wind Turbine Anomaly Detection
+
+This sample app demonstrates a time series use case by detecting anomalous power generation patterns
+in wind turbines, relative to wind speed. By identifying deviations, it helps
+optimize maintenance schedules and prevent potential turbine failures, enhancing
+operational efficiency.
+
+## App Architecture
+
+As seen in the following architecture diagram, the sample app at a high-level comprises of data simulators(can act as data destinations if configured) - these in the real world would be the physical devices, the generic Time Series AI stack based on **TICK Stack** comprising of Telegraf, InfluxDB, Time Series Analytics microservice using Kapacitor and Grafana.
+
+![Time Series AI Stack Architecture Diagram](../_images/time-series-ai-stack-architecture.png)
```

Comment:
Same as above, this line change can be avoided. I think this file is already up-to-date.